cookbooks/pypy-home/templates/default/nginx.conf.erb

```diff
@@ -1,11 +1,21 @@
 server {
   listen <%= @resource.port %>;
   server_name <%= @resource.server_name.is_a?(Array) ? @resource.server_name.join(' ') : @resource.server_name %>;
-  <% if @resource.ssl %>
-  ssl on;
+
+<% if @resource.ssl %>
+  return 301 https://$server_name$request_uri;
+}
+
+server {
+  listen 443 ssl;
+  server_name <%= @resource.server_name.is_a?(Array) ? @resource.server_name.join(' ') : @resource.server_name %>;
+
   ssl_certificate <%= @resource.ssl_certificate %>;
   ssl_certificate_key <%= @resource.ssl_certificate_key %>;
-  <% end %>
+
+  add_header Strict-Transport-Security "max-age=31536000";
```

Comment:
Hi @alex! :)

I'd qualify this also with an `always` so it is returned on non-200 status codes as well.

So just:

```
add_header Strict-Transport-Security "max-age=31536000" always;
```